src/screens/ChannelListScreenV2/index.tsx

```diff
@@ -1,166 +1,72 @@
-import {fcmTokenService} from '../../service/users';
-import StorageUtils from '../../utils/storage';
-import ChannelListScreen from '../ChannelListScreen';
-import Search from '../ChannelListScreen/elements/Search';
-import AnonymousChannelListScreen from './AnonymousChannelListScreen';
-import {
-  PERMISSION_STATUS_ACCEPTED,
-  PERMISSION_STATUS_BLOCKED,
-  PERMISSION_STATUS_PENDING
-} from '../../utils/constants';
 
-const ChannelListScreenV2 = () => {
+const ChannelListScreenV2 = ({route}) => {
   const {refresh} = useLocalDatabaseHook();
+  const {checkNotificationPermission} = useRootChannelListHook();
   const navigation = useNavigation();
-  const {profile} = useUserAuthHook();
   const isFocused = navigation.isFocused();
-
-  const [selectedTab, setSelectedTab] = React.useState(0);
-  const {signedChannelUnreadCount, anonymousChannelUnreadCount} = useRootChannelListHook();
-
-  const navigateToContactScreen = () => {
-    navigation.navigate('ContactScreen');
-  };
-
-  const requestPermission = async () => {
-    const authStatus = await messaging().requestPermission();
-    const enabled =
-      authStatus === messaging.AuthorizationStatus.AUTHORIZED ||
-      authStatus === messaging.AuthorizationStatus.PROVISIONAL;
-
-    if (enabled) {
-      const fcmToken = await messaging().getToken();
-      const payload = {
-        fcm_token: fcmToken
-      };
-      fcmTokenService(payload);
-    }
-  };
+  const {
+    channels: anonChannels,
+    goToChatScreen,
+    goToPostDetailScreen,
+    goToContactScreen
+  } = useAnonymousChannelListScreenHook();
 
   React.useEffect(() => {
-    const checkNotif = async () => {
-      if (!navigation.isFocused()) return;
-      PushNotification.checkPermissions(
-        (
-          data: PushNotificationPermissions & {
-            authorizationStatus?: number;
-          }
-        ) => {
-          const lastPromptTime = StorageUtils.lastPromptNotification.get();
-          const promptInterval = 48 * 60 * 60 * 1000;
-          const currentTime = new Date().getTime();
-          if (lastPromptTime && currentTime - parseFloat(lastPromptTime) < promptInterval) {
-            return; // Don't show the prompt if the interval has not passed
-          }
-
-          const needsPermission =
-            data.alert === false || data.badge === false || data.sound === false;
+    if (isFocused) {
+      refresh('channelList');
+      checkNotificationPermission();
+    }
+  }, [isFocused]);
 
-          const showAlert = (onPressAction: () => void) =>
-            Alert.alert(
-              "Don't Miss New Messages",
-              `Allow notifications to know 
-when friends send you messages.`,
-              [
-                {text: 'Not now', onPress: () => console.log('Cancel Pressed')},
-                {text: 'Allow', onPress: onPressAction, isPreferred: true}
-              ]
-            );
+  const renderChannelItem = ({item}) => {
+    if (item?.channelType === 'ANON_PM') {
```

Comment:
Should be using constant variable for `ANON_PM`

Reply:
solved in new commit